adal/src/androidTest/java/com/microsoft/aad/adal/Util.java

```diff
@@ -91,7 +91,19 @@ static TokenCacheItem getTokenCacheItem(final String authority, final String res
         
         return tokenCacheItem;
     }
-    
+
+    static String getSuccessTokenResponseWithFamilyClientId() {
```

Comment:
I would consider cleaning this up using `JSONObject` -- since this JSON is currently pretty simple and this is only test code I wouldn't let this stop ship. That said, I'd much prefer to see something like the following

```
JSONObject json = new JSONObject();
json.putString("id_token", TEST_IDTOKEN);
json.putString("access_token", "blah blah");
...
return json.toString();
```

Reply:
can do it.